internal/website/themes/hyde/static/css/hyde.css

```diff
@@ -70,6 +70,11 @@ html {
   }
 }
 
+figure.aligncenter img {
```

Comment:
Are there issues with aligning the figure element? I would expect the `figure` element to be block-level, whereas `img` is typically inline.

Reply:
Sorry, I'm not sure I'm following. Can you please elaborate?

For reference, the HTML created by Hugo for:

```
{{< figure src="go-cdk-logo-gopherblue-small.png" alt="Go CDK logo"
class="aligncenter" >}}
```

Is:


```
<figure class="aligncenter">
    <img src="go-cdk-logo-gopherblue-small.png"
         alt="Go CDK logo"/> 
</figure>
```

Reply:
Sure. I'm wondering why the rule selector is targetting the `img` element and not the `figure` element (and also suggesting that perhaps the rule selector should target the `figure` element).